src/utils/db.ts

```diff
@@ -480,14 +480,7 @@ export const updateLinkedGoalsByProject = async (projectId: string, activityId:
                 // mark criterias with these goals as deleted
                 const updateCriterias = ctx.goalAchieveCriteria.updateMany({
                     where: {
-                        OR: [
-                            {
-                                criteriaGoalId: { in: goalIds.map(({ id }) => id) },
```

Comment:
It will cause prod to 500 if it's still used in current release

Reply:
Oh, well needs create one more pr

Reply:
https://github.com/taskany-inc/issues/pull/2135

Reply:
Now, this pr contain change only in trpc procedures